modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/Bar/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ActivationKeysActionsButton from '../ActionButton';
+
+const ActivationKeysManagementBar = ({accountKey, sessionId}) => {
+	return (
+		<div className="align-items-center bg-neutral-1 d-flex justify-content-between mt-4 px-3 py-3 rounded">
+			<div>
+				<ActivationKeysActionsButton
+					accountKey={accountKey}
+					sessionId={sessionId}
+				/>
+			</div>
```

Comment:
This `<div>` is necessary?

```suggestion
			<ActivationKeysActionsButton
					accountKey={accountKey}
					sessionId={sessionId}
				/>
```

Reply:
Yes, without this div, when alerts is showed, it creates a div before the button tag and push him to right side